src/sagemaker_inference/environment.py

```diff
@@ -35,6 +35,7 @@
     model_dir = os.path.join(base_dir, "model")  # type: str
 # str: the directory where models should be saved, e.g., /opt/ml/model/
 
+config_dir = os.path.join(base_dir, "etc")  # type: str
```

Comment:
Naming this directory etc makes sense to me because the source code of sagemaker-inference-toolkit also has a [etc](https://github.com/aws/sagemaker-inference-toolkit/tree/master/src/sagemaker_inference/etc) directory containing similar config files.